TurkishFilter/sections/specific.txt

```diff
@@ -85,6 +85,7 @@ dizifilmix.org,sineturk.pw,altyazilifilmizle.tv,1080pfilmizle.net,dolufilmizle.c
 bandirmahaber.com.tr##.th-addbox
 bandirmahaber.com.tr##.th-postdetail > center > a[href^="tel:"] > img
 ||bandirmahaber.com.tr/nakliye.jpg
+kulturmafyasi.com##footer > div.cb-post-large
```

Comment:
Not shown in code for me.

Reply:
<details><summary>Screenshot:</summary>

![kultur](https://user-images.githubusercontent.com/33488229/66644285-35af3580-ec29-11e9-9afb-b6b33ef94a7b.png)
</details><br/>